cpp/src/arrow/record_batch_test.cc

```diff
@@ -682,7 +682,8 @@ TEST_F(TestRecordBatch, ToTensorEmptyBatch) {
   ASSERT_OK_AND_ASSIGN(std::shared_ptr<RecordBatch> empty,
                        RecordBatch::MakeEmpty(schema));
 
-  ASSERT_OK_AND_ASSIGN(auto tensor, empty->ToTensor());
+  ASSERT_OK_AND_ASSIGN(auto tensor,
+                       empty->ToTensor(/*null_to_nan=*/false, /*row_major=*/false));
   ASSERT_OK(tensor->Validate());
 
   const std::vector<int64_t> strides = {4, 4};
```

Comment:
The strides wouldn't be different here for row major, I suppose? Might be good to explicitly test both